macos/FluentUITestApp/FluentUITestApp-Info.plist

```diff
@@ -17,9 +17,9 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.10.0</string>
+	<string>0.10.1</string>
 	<key>CFBundleVersion</key>
-	<string>55</string>
+	<string>62.10.1</string>
```

Comment:
what happen here?

Reply:
Looks like I missed it in 0.10.0, and it wasn't caught in the PR.